CMakeLists.txt

```diff
@@ -18,6 +18,7 @@
 cmake_minimum_required(VERSION 3.28)
 project(raw_pointers_safe_deletion_tests)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Notice that I'm updating the C++ standard in use here. I think Roy's been wanting to do that for a while. I am now at the point where I am itching to update, as well.

Reply:
Also, the `owner<... *>` construct apparently won't work unless we update to C++14. I think it's time. Just about everything we support should be compatible by now.